resources/dicts/patrols/beach/hunting/any.json

```diff
@@ -120,113 +92,55 @@
+        "decline_text": "app1 asks {PRONOUN/app1/poss} mentor to do the assessment some other time.",
         "chance_of_success": 40,
         "exp": 20,
         "success_text": [
             "app1 manages a skillful catch, and brings it back to the camp to show everyone!",
-            "app1 focuses completely on their test, slipping into a clear-eyed, razor focused routine of catch and stash, catch and stash. Eventually they've caught enough to need their mentor's help carrying it back to camp, and the feeling seeing their prey stock the fresh-kill pile - they feel ready to burst with happiness!",
+            "app1 focuses completely on {PRONOUN/app1/poss} test, slipping into a clear-eyed, razor focused routine of catch and stash, catch and stash. Eventually {PRONOUN/app1/subject}{VERB/app1/'ve/'s} caught enough to need {PRONOUN/app1/poss} mentor's help carrying it back to camp, and the feeling seeing {PRONOUN/app1/poss} prey stock the fresh-kill pile - {PRONOUN/app1/subject} {VERB/app1/feel/feels} ready to burst with happiness!",
             null,
-            "It's a terrible day for hunting, the prey running sparse and alert. But s_c takes their time and doesn't let the conditions discourage them, until eventually the perfect opportunity falls into place and they pull off a perfect catch!"
+            "It's a terrible day for hunting, the prey running sparse and alert. But s_c takes {PRONOUN/s_c/poss} time and doesn't let the conditions discourage them, until eventually the perfect opportunity falls into place and {PRONOUN/s_c/subject} {VERB/s_c/pull/pulls} off a perfect catch!"
         ],
         "fail_text": [
             "Hunting is poor, and while app1's mentor is disappointed, they don't expect r_c to be able to catch what isn't there.",
-            "s_c is too hasty, bouncing after first one prey, then another, and it ends with them empty pawed and exhausted. Their mentor is disappointed - they clearly need more training.",
-            null,
-            "While on the hunt, app1 tumbles right down into a rock pool while their attention is elsewhere. Bruised and sore, r_c has to report to the medicine cat den when they return to camp."
-        ],
-        "win_trait": [
-            "calm",
-            "careful",
-            "insecure",
-            "lonesome",
-            "loyal",
-            "nervous",
-            "sneaky",
-            "strange",
-            "patient",
-            "thoughtful",
-            "wise"
-        ],
-        "fail_trait": [
-            "adventurous",
-            "bold",
-            "charismatic",
-            "childish",
-            "confident",
-            "daring",
-            "playful",
-            "righteous"
+            "s_c is too hasty, bouncing after first one prey, then another, and it ends with {PRONOUN/app1/object} empty pawed and exhausted. {PRONOUN/app1/poss/CAP} mentor is disappointed - {PRONOUN/app1/subject} clearly {VERB/app1/need/needs} more training.",
```

Comment:
should be tagging 's_c' in the PRONOUN tags, for consistency

Reply:
OOPS gotcha!